src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -23,318 +23,326 @@ import * as PolicyUtils from '@libs/PolicyUtils';
+    reportID?: string;
 
     /** File path of the receipt */
-    receiptPath: PropTypes.string,
+    receiptPath?: string;
 
     /** File name of the receipt */
-    receiptFilename: PropTypes.string,
+    receiptFilename?: string;
 
     /** List styles for OptionsSelector */
-    listStyles: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.object), PropTypes.object]),
+    listStyles?: StyleProp<ViewStyle>;
 
     /** ID of the transaction that represents the money request */
-    transactionID: PropTypes.string,
+    transactionID?: string;
 
     /** Transaction that represents the money request */
-    transaction: transactionPropTypes,
-
-    /** Unit and rate used for if the money request is a distance request */
-    mileageRate: PropTypes.shape({
-        /** Unit used to represent distance */
-        unit: PropTypes.oneOf([CONST.CUSTOM_UNITS.DISTANCE_UNIT_MILES, CONST.CUSTOM_UNITS.DISTANCE_UNIT_KILOMETERS]),
-
-        /** Rate used to calculate the distance request amount */
-        rate: PropTypes.number,
-
-        /** The currency of the rate */
-        currency: PropTypes.string,
-    }),
+    transaction?: OnyxEntry<OnyxTypes.Transaction>;
 
     /** Whether the money request is a distance request */
-    isDistanceRequest: PropTypes.bool,
+    isDistanceRequest?: boolean;
 
     /** Whether the money request is a scan request */
-    isScanRequest: PropTypes.bool,
+    isScanRequest?: boolean;
 
     /** Whether we're editing a split bill */
-    isEditingSplitBill: PropTypes.bool,
+    isEditingSplitBill?: boolean;
 
     /** Whether we should show the amount, date, and merchant fields. */
-    shouldShowSmartScanFields: PropTypes.bool,
+    shouldShowSmartScanFields?: boolean;
 
     /** A flag for verifying that the current report is a sub-report of a workspace chat */
-    isPolicyExpenseChat: PropTypes.bool,
+    isPolicyExpenseChat?: boolean;
 
-    /* Onyx Props */
-    /** Collection of categories attached to a policy */
-    policyCategories: PropTypes.objectOf(categoryPropTypes),
-
-    /** Collection of tags attached to a policy */
-    policyTags: tagPropTypes,
+    /** Whether smart scan failed */
+    hasSmartScanFailed?: boolean;
 
-    /* Onyx Props */
-    /** The policy of the report */
-    policy: policyPropTypes.policy,
-
-    /** Holds data related to Money Request view state, rather than the underlying Money Request data. */
-    iou: iouPropTypes,
+    /** The ID of the report action */
+    reportActionID?: string;
 };
 
-const defaultProps = {
-    onConfirm: () => {},
-    onSendMoney: () => {},
-    onSelectParticipant: () => {},
-    iouType: CONST.IOU.TYPE.REQUEST,
-    iouCategory: '',
-    iouTag: '',
-    iouIsBillable: false,
-    onToggleBillable: () => {},
-    payeePersonalDetails: null,
-    canModifyParticipants: false,
-    isReadOnly: false,
-    bankAccountRoute: '',
-    session: {
-        email: null,
+function MoneyRequestConfirmationList({
+    transaction,
```

Comment:
This way we can get rid of `?? null` in several places in this file, wdyt?
```suggestion
    transaction = null,
```